controllers/users.js

```diff
@@ -221,7 +221,12 @@ const updateSelf = async (req, res) => {
       await userQuery.setIncompleteUserDetails(userId);
     }
 
-    const user = await userQuery.addOrUpdate(req.body, userId);
+    const data = {
+      status: "ACTIVE",
```

Comment:
Why `status` is set to **ACTIVE** and not to **IDLE** here?

Reply:
waiting for reply for this message: https://discord.com/channels/673083527624916993/1069806199853285426/1097392142252515388

As discussed in past with @bhtibrewal https://github.com/Real-Dev-Squad/website-backend/issues/990#issuecomment-1505473503

Reply:
after discussion with ankush and bhavika
you can check the message
<img width="577" alt="image" src="https://user-images.githubusercontent.com/107163260/233054856-53e8c2bd-6546-40ec-86a3-059e3536d374.png">